game-core/src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -441,24 +435,13 @@ ServerGame getIGame() {
     return game;
   }
 
-  private static synchronized void restartLobbyWatcher(
-      final SetupPanelModel setupPanelModel, final ServerGame serverGame) {
+  private synchronized void restartLobbyWatcher() {
     try {
-      final ISetupPanel setup = setupPanelModel.getPanel();
-      if (setup == null) {
+      final HeadlessServerSetup setup = setupPanelModel.getPanel();
+      if (setup == null || game != null || setup.canGameStart()) {
```

Comment:
I personally like clarifying parens, it's an IDEA analysis setting that can be turned on, I'm pretty sure Eclipse warnings would have the same (Eclipse turns out to have quite a few if they are throttle up beyond the default, but you knew that already :+1: )

This specific nit is pretty off-topic, the cleanup here is pretty sweet

Reply:
I can see that they can enhance readability for more complex if cases, but in this case I don't think they are necessary.